README.md

```diff
@@ -6,9 +6,25 @@
 
 [![Coverage Status](https://coveralls.io/repos/github/zooniverse/front-end-monorepo/badge.svg?branch=master)](https://coveralls.io/github/zooniverse/front-end-monorepo?branch=master)
 
+**Table of Contents**
+
+- [Requirements](#requirements)
+- [Monowhat?](#monowhat)
+- [Getting started](#getting-started)
+- [Helpful Guides](#helpful-guides)
+- [Packages](#packages)
+- [Helpers](#helpers)
+- [Conventions](#conventions)
+  - [NPM](#npm)
+  - [Packages directory](#packages-directory)
+- [Production deployment](#production-deployment)
+- [License](#license)
+
 ## Requirements
 
-[Browser support](docs/arch/adr-3.md)
+- [Browser support](docs/arch/adr-3.md)
+- Node 8
```

Comment:
Node 10 is the LTS now. Maybe this should be >=8?

Reply:
Sure, but we're not using it anywhere; I'm happy to do a separate PR to bump to 10 across the repo.

Reply:
I'm ok with separate update PR. Thanks.

Reply:
cc #595